pkgs/tools/text/rsbkb/default.nix

```diff
@@ -6,16 +6,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rsbkb";
-  version = "1.4";
+  version = "1.4_823b35";
 
   src = fetchFromGitHub {
-    owner = "trou";
+    # NOTE: patched until next release for https://github.com/NixOS/nixpkgs/issues/332957
+    # owner = "trou";
+    owner = "ProducerMatt";
     repo = "rsbkb";
-    rev = "release-${version}";
-    hash = "sha256-c5+Q/y2tZfhXQIAs1W67/xfB+qz1Xn33tKXRGDAi3qs=";
+    # rev = "release-${version}";
+    rev = "823b35072d60d1526c5646bd850a39f056c3cfc7";
+    hash = "sha256-tc0nOx8Z8ytH0AS9nOhk/0ejBEKcg0lUF289TL1iRHM=";
   };
```

Comment:
It may be better to keep the original `src` but add your diff to `cargoPatches`.

Reply:
Oh darn I missed this. In the docs, `cargoLock.lockFileContents` is presented first and `cargoPatches` is presented later in another section.